R/utilities.R

```diff
@@ -7,6 +7,27 @@ displayNames <- function(proj) { # create unique names for each sequence file
     sprintf(paste("s%0", ndigits, "i_%s", sep = ""), seq_along(samples), samples)
 }
 
+#' Internal function to get seqlengths, mapped and unmapped counts for a bam file
+#' 
+#' @param bf Character scalar with the path and filename to a bam file (index
+#'     is assumed to be already existing)
+#' 
+#' @return a named numeric vector with elments 'seqlength', 'mapped' and 'unmapped'
```

Comment:
elments -> elements

Reply:
Thank you, I fixed the typo and have also re-activated the GHA workflow, so that now we should also see the check results before I will merge and push to upstream.